app.rb

```diff
@@ -4,6 +4,10 @@
 # IAM - a resource usage metric collection and reporting system
 class Iam < Sinatra::Base
   set :show_exceptions, :after_handler
+  enable :method_override
+  set :port, 4567
```

Comment:
yeah after reading your comments, this is something environments.rb should have and not app.rb